astropy/units/tests/test_format.py

```diff
@@ -542,15 +542,13 @@ def test_vounit_custom():
     assert x_string == 'm mfoo'
 
 
-def test_vounit_implicit_custom():
-    # Yikes, this becomes "femto-urlong"...  But at least there's a warning.
-    with pytest.warns(UnitsWarning) as w:
-        x = u.Unit("furlong/week", format="vounit")
-    assert x.bases[0]._represents.scale == 1e-15
-    assert x.bases[0]._represents.bases[0].name == 'urlong'
-    assert len(w) == 2
-    assert 'furlong' in str(w[0].message)
-    assert 'week' in str(w[1].message)
+def test_vounit_no_more_implicit_custom():
+    # This becomes UnrecognizedUnit(furlong/week)
+    with pytest.warns(UnitsWarning, match="'furlong/week' did not parse as vounit unit") as w:
+        x = u.Unit("furlong/week", format="vounit", parse_strict='warn')
+    assert isinstance(x, u.UnrecognizedUnit)
```

Comment:
If I understood what #8978 says then the fact that this is now an `UnrecognizedUnit` would be a violation of the VOUnit standard.

Reply:
Oh, boy. Thanks for digging up all the old issues/PRs that I have no recollection of participating in even though apparently I did. Looks like another attempt was done in #8979 but failed.

So, yeah, maybe this is not the correct solution even though it seems desirable for `astroquery`. I wonder if @mhvk , @tomdonaldson , or @dhomeier have any new ideas.